google/cloud/forseti/scanner/audit/bigquery_rules_engine.py

```diff
@@ -70,8 +70,7 @@ def build_rule_book(self, global_configs=None):
         self.rule_book = BigqueryRuleBook(self._load_rule_definitions())
 
     # TODO: The naming is confusing and needs to be fixed in all scanners.
```

Comment:
all #TODO items related to renaming are deleted. Please note that I deleted the #TODO items along with the \n line change at the end of each line. Because of this, the function definition in the next line appears to be changed in the comparison, but it is actually only moved one line up.